.golangci.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+
+run:
+  timeout: 3m
+
+formatters:
+  enable:
+    - goimports
+
+linters:
+  disable:
+    - errcheck
+  enable:
+    - forbidigo
+    - gocritic
+    - lll
+    - reassign
+    - unconvert
+  settings:
+    gocritic:
+      disabled-checks:
+        - ifElseChain
+    lll:
+      line-length: 100
+      tab-width: 4
```

Comment:
I would prefer stricter checks. They simplify writing code and reviewing.

Please, add more linters to make the rules more strict, as example:

https://github.com/tarantool/go-discovery/blob/master/.golangci.yml

You could also add linters on your flavor.

Reply:
Why not do it now?